### ISSUE TITLE
Batch size makes no difference in training

### ISSUE THREAD

**Issue by sajastu** [yuanxiaosc/BERT-for-Sequence-Labeling-and-Text-Classification#8]
Hello,

Just wondering how batch size in this implementation of sequence labeling works. I'm trying different batch sizes for both train and validation sets, but nothing literally changes. Like, the memory that bert takes on GPUs is no more than 135MiB.

![image](https://user-images.githubusercontent.com/10419055/64914290-d985ee00-d71c-11e9-8d5f-b7f1735d86c1.png)

Could you explain what's going on in there?
Thanks!

**Comment by yuanxiaosc** [comment]
@sajastu This code simply modifies the model output of the BERT and combines the classification tasks with the sequence labels. Because they all experiment on small datasets, all models take up less space, just like the BERT demo demo https://github.com/google-research/bert. I have not done experiments on the GPU, the memory occupied in the CPU is about 50G.
